scripts/check-image-version.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# ############
+# This script will check  the version of the COMPONENT.WAR/META-INF/MANIFEST.mf file by comparing the
+# value of the "Implementation-Version" field which would match with the KIE_VERSION from the build properties
+# file for some components.
+#
+# This script can be used to check the version the following Components:
+#   - Dashbuilder
+#   - KIE Server
+#   - Controller
+#   - Business Central Monitoring
+#   - Business Central
+#   - Smartrouter
+#
+# For Smartrouter, it is needed to extract the rhpam-7.13.x-smart-router.jar file to read this information
+# from META-INF.
+#
+# For Business Central, the version is retrieved from META-INF/build.metadata
+# For Process Instance Migration
+#
+# ############
+# Usage:
+# bash check-image-version.sh $COMPONENT_IMAGE $KIE_VERSION
+# e.g.:
+#     bash check-image-version.sh @rhpam-7/rhpam-dashbuilder-rhel8 7.67.0.Final-redhat-00005
```

Comment:
Shouldn't the run example be: 
`bash check-image-version.sh rhpam-dashbuilder-rhel8 7.67.0.Final-redhat-00005`

without the `@rhpam-7/` string.

Reply:
agreed, removed.